themes/bootstrap3/templates/myresearch/deleteaccount.phtml

```diff
@@ -0,0 +1,20 @@
+<h2><?=$this->transEsc('delete_account_title') ?></h2>
+<?=$this->flashmessages();?>
+<? if ($this->accountDeleted): ?>
+  <div id="delete-account-success">
+    <p><?=$this->translate('delete_account_success_message'); ?></p>
+    <?
+    // Logout redirect with inline script to make it lightbox compatible
+    $script = "setTimeout(function() { window.location = '{$this->redirectUrl}'; }, 3000);";
```

Comment:
I wonder if @crhallberg has an alternative solution to this problem...

Reply:
Would be nice of course, but note that the delay there is also intentional.

Reply:
This is the simplest and clearest solution I can think of. Makes sense to redirect rather than just close the lightbox.